lib/collections/collections.js

```diff
@@ -113,6 +113,14 @@ export const Packages = new Mongo.Collection("Packages");
 
 Packages.attachSchema(Schemas.PackageConfig);
 
+/**
+ * Catalog Collection
+ * @ignore
+ */
+export const Catalog = new Mongo.Collection("Catalog");
+
+// Catalog.attachSchema(Schemas.Catalog);
```

Comment:
Can we remove this commented code?

Reply:
will remove, and add a TODO to attach a schema.